src/openrct2/scenario/Scenario.cpp

```diff
@@ -237,8 +237,8 @@ void scenario_success_submit_name(const char* name)
  */
 static void scenario_entrance_fee_too_high_check()
 {
-    money16 totalRideValueForMoney = gTotalRideValueForMoney;
-    money16 max_fee = totalRideValueForMoney + (totalRideValueForMoney / 2);
+    const money16 totalRideValueForMoney = gTotalRideValueForMoney;
+    const money32 max_fee = static_cast<money32>(totalRideValueForMoney) + (totalRideValueForMoney / 2);
```

Comment:
Use the clamp function

Reply:
I get where that suggestion is coming from, but is it really a good idea especially if you're moving away from `money16`? Seems to me like letting the value be sufficiently big is a good solution.

Reply:
When we’re moving away from money16, we’ll also replace all instances of `add_clamp_money16()` ;-)

Reply:
There is no `add_clamp_money16` function - I will add one.